casper/src/test/scala/coop/rchain/casper/genesis/contracts/StandardDeploysSpec.scala

```diff
@@ -0,0 +1,16 @@
+package coop.rchain.casper.genesis.contracts
+
+import coop.rchain.crypto.codec.Base16
+import org.scalatest.FlatSpec
+
+class StandardDeploysSpec extends FlatSpec {
```

Comment:
What's the intention on this test?

Reply:
Just to record in the test logs what are the corresponding public keys.